lando/server/tests/test_cwlworkflow.py

```diff
@@ -89,8 +94,8 @@ def test_simple_workflow_missing_files(self):
             cwl_base_command = [
                 "cwl-runner",
                 "--debug",
-                "--tmpdir-prefix=/Users/jpb67",
-                "--tmp-outdir-prefix=/Users/jpb67",
+                "--tmpdir-prefix=" + USER_TEMP_DIR,
```

Comment:
Why are these arguments necessary? I thought the tmpdir-prefix/tmp-outdir-prefix were only needed when the workflow steps would run inside docker?